### PR TITLE
Cloudnugetissue

### PR DESCRIPTION
**Pull request by janstaelensskyline**

### CODE REVIEW

File: .github/workflows/Automation Master Legacy Workflow.yml [hunk 1]
```diff
@@ -123,9 +123,19 @@ jobs:
         uses: microsoft/setup-msbuild@v2
         
       - name: Enable Skyline GitHub NuGet Registry
```
Comment (MichielOda) [inline .github/workflows/Automation Master Legacy Workflow.yml:125]: Add a TODO to refactor this with a loop the next time we need to make changes to these steps.

File: .github/workflows/Automation Master Legacy Workflow.yml [hunk 2]
```diff
@@ -299,25 +310,38 @@ jobs:
         run: |
           echo solutionFilePath=$(find . -type f -name '*.sln') >> $GITHUB_OUTPUT
         shell: bash
-
-      - name: Find or create NuGet.config file
-        if: inputs.owner == 'SkylineCommunications'
-        id: findOrCreateNuGetFile
+    
+      - name: Find branch
```
Comment (MichielOda) [inline .github/workflows/Automation Master Legacy Workflow.yml:314]: Merge/Rebase main branch in this branch as currently there are conflicts and the PR contains changes that were also done in main.

**Review by MichielOda — CHANGES_REQUESTED** [review]
Don't forget to update the connector workflow, NuGet workflows & DotNet tool workflows

File: .github/workflows/Automation Master Legacy Workflow.yml [hunk 3]
```diff
@@ -123,9 +123,19 @@ jobs:
         uses: microsoft/setup-msbuild@v2
         
       - name: Enable Skyline GitHub NuGet Registry
-        if: inputs.owner == 'SkylineCommunications'
```
[the remark this answers is not in the document]
Comment (janstaelensskyline) [reply .github/workflows/Automation Master Legacy Workflow.yml:126]: ok

File: .github/workflows/Automation Master Legacy Workflow.yml [hunk 4]
```diff
@@ -299,25 +310,38 @@ jobs:
...
+          # Append to GitHub Actions output
+          echo "branch=${branches}" >> $GITHUB_OUTPUT
+        shell: bash 
+
+      - name: Target Branch
+        id: showResult
+        run: echo "${{ steps.findBranch.outputs.branch }}"
+
       - name: Enable Skyline GitHub NuGet Registry
-        if: inputs.owner == 'SkylineCommunications'
```
[the remark this answers is not in the document]
Comment (janstaelensskyline) [reply .github/workflows/Automation Master Legacy Workflow.yml:335]: ok

File: .github/workflows/Automation Master SDK Workflow.yml [hunk 5]
```diff
@@ -76,8 +76,19 @@ jobs:
         shell: bash
 
       - name: Enable Skyline GitHub NuGet Registry
-        if: inputs.owner == 'SkylineCommunications'
```
[the remark this answers is not in the document]
Comment (janstaelensskyline) [reply .github/workflows/Automation Master SDK Workflow.yml:79]: ok

File: .github/workflows/Automation Master Legacy Workflow.yml (hunk 1 above)
Comment (janstaelensskyline) [reply .github/workflows/Automation Master Legacy Workflow.yml:125]: ok

File: .github/workflows/Automation Master SDK Workflow.yml [hunk 6]
```diff
@@ -202,8 +215,9 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
+
+      - name: readDir
+        run: ls
```
Comment (MichielOda) [inline .github/workflows/Automation Master SDK Workflow.yml:220]: readDir can be removed as it doesn't serve any purpose. 
Fetch-depth need to be readded again as that is needed for SonarCloud.

File: .github/workflows/Automation Master SDK Workflow.yml [hunk 7]
```diff
@@ -311,7 +323,7 @@ jobs:
   
       - name: Upload to Catalog
         id: uploadToCatalog
-        run: echo "id=$(dataminer-catalog-upload with-registration --path-to-artifact "${{ github.workspace }}/${{ steps.packageName.outputs.name }}.dmapp" --uri-sourcecode "${{ github.server_url }}/${{ github.repository }}" --artifact-version ${{ inputs.referenceName }} --branch "${{ steps.findBranch.outputs.branch }}" --dm-catalog-token ${{ secrets.api-key }})" >> $GITHUB_OUTPUT
+        run: echo "id=$(dataminer-catalog-upload with-registration --path-to-artifact "${{ github.workspace }}/${{ steps.packageName.outputs.name }}.dmapp" --uri-sourcecode "${{ github.server_url }}/${{ github.repository }}" --artifact-version ${{ inputs.referenceName }} --dm-catalog-token ${{ secrets.api-key }})" >> $GITHUB_OUTPUT
```
Comment (MichielOda) [inline .github/workflows/Automation Master SDK Workflow.yml:326]: This shouldn't be removed

**Review by MichielOda — CHANGES_REQUESTED** [review]

File: .github/workflows/Automation Master SDK Workflow.yml (hunk 6 above)
Comment (janstaelensskyline) [reply .github/workflows/Automation Master SDK Workflow.yml]: ok

File: .github/workflows/Automation Master SDK Workflow.yml (hunk 7 above)
Comment (janstaelensskyline) [reply .github/workflows/Automation Master SDK Workflow.yml]: ok

**Review by MichielOda — APPROVED** [review]

**Pull request merged**